app/react-native/src/preview/index.js

```diff
@@ -133,15 +135,22 @@ export default class Preview {
     channel.emit(Events.GET_CURRENT_STORY);
   }
 
-  _setInitialStory = () => {
-    const story = this._getInitialStory();
+  _setInitialStory = async (shouldPersistSelection: boolean) => {
```

Comment:
I may be off but since when JS supports type definitions?

Reply:
oops. good catch. 🙄